src/components/header/Header.js

```diff
@@ -0,0 +1,18 @@
+import './header.css';
+import Logo from '../logo/Logo';
+import Nav from '../navigation/nav';
+import Phone from '../phone/Phone';
+
+function Header() {
+  return (
+      <header class="header wrap">
```

Comment:
it's recommended to use className instead of class in JSX. Fix this everywhere
https://ru.reactjs.org/docs/dom-elements.html#classname